vuepress/docs/next/docs/reference/databases.md

```diff
@@ -347,3 +347,13 @@ After updating parameters with the one you need, you should end with a yaml like
 Using `spec.dbms: "none"` will cause the operator to skip that initial schema/user creation step entirely.
 Then adding those variables under the `spec.environmentVariables` section will supply connection parameters that will be used by EntandoApp.
 Keep in mind that all these parameters will be applied to each of the containers in the EntandoApp pod and that they will also override existing values.
+
+## Liquibase Migration
+
+Beginning with Entando 7.0, AppEngine (DeApp) modules will implement automatic Liquibase migrations to manage structural changes to databases running on MySQL or PostgreSQL.
```

Comment:
Maybe we should just say Entando App Engine instead of AppEngine (DeApp). I think that's what's in the glossary and other pages - https://developer.entando.com/v6.3.2/docs/getting-started/concepts-overview.html#entando-app-engine